gravitee-apim-rest-api/postman/README.adoc

```diff
@@ -36,25 +37,11 @@ newman run postman/test/collections/[...]collection.json -e postman/env/Gravitee
 
 
 == Run all the tests collections locally using shell script.
-```
-sh scripts/tests.sh
-```
-
-== Run all the tests collections on Nightly or Demo environment
-
-*Nightly*
 
-```
-sh scripts/tests.sh --postman-env=nightly --postman-health-url=nightly.gravitee.io/api/management/swagger.json
-
-```
-
-*Demo*
-
-```
-sh scripts/tests.sh --postman-env=demo --postman-health-url=demo.gravitee.io/management/swagger.json
-
-```
+[source, shell]
+----
+sh scripts/tests.sh
```

Comment:
I would just have let the `--postman-health-url` in this documentation to know we can use what we want to consider the app as health

Reply:
Done